### ISSUE TITLE
Can't publish large image file (< 2MB)

### ISSUE THREAD

**Issue by marcaddeo** [cloudflare/wrangler#775]
### 🐛 Bug Report

When using `wrangler publish` I get the following error trying to upload a 1.7MB gif file to the KV:

```
Error: File `guides/devilhunter-mod/images/remove-end-plate.07cca7649024261146dbf943c6f4f238e463c1f48d5d89a30083c8dca560d8ab.gif` of 2265836 bytes exceeds the maximum value size limit of 2097152 bytes
```

But if I attempt to upload the image via the KV control panel in the Cloudflare dashboard, I don't get the error and it uploads without any issue.

#### Environment

* operating system: `macOS 10.13.6`
* output of `rustc -V`: `rustc 1.38.0 (625451e37 2019-09-23)`
* output of `node -v`: `v12.11.1`
* output of `wrangler -V`:  `✨  wrangler 1.4.0`

#### Steps to reproduce

Create a 1.7MB file:
`head -c 1699377 /dev/urandom > test.gif`

And then run `wrangler publish`

#### What did you expect to see?

No errors

#### What did you see instead?
```
Error: File `guides/devilhunter-mod/images/test.650921fd9fdbfd3866aedad4d4f191ac109b68a1f0053a0e3157950b929a6126.gif` of 2265836 bytes exceeds the maximum value size limit of 2097152 bytes
```

**Comment by gabbifish** [comment]
Hi there! Workers KV, as of last week, should now support 10MB files--please try again and let me know if it works. :) Sorry for the inconvenience!

**Comment by gabbifish** [comment]
As for the inconsistency between the UI and Wrangler, we believe we know the source of this problem--it looks like wrangler performs size checks on items after they have been base64 encoded (which explains why a smaller file looks larger to wrangler; base64 encoding is longer than raw bytes). We'll get a fix in for 1.5.0.

**Comment by marcaddeo** [comment]
@gabbifish thanks for the fix! I haven't had a moment to test it, but will let you know if it works once I do.

**Comment by ashleymichal** [comment]
to be clear, we need to make the appropriate adjustments to Wrangler, which still asserts the 2MB limit. the fix will be added to the 1.5.0 milestone.

**Comment by marcaddeo** [comment]
@ashleymichal gotcha. I shall not test it then!
